.github/workflows/frontend_runtime_e2e_trigger_response.yml

```diff
@@ -96,7 +99,13 @@ jobs:
   e2e_run:
     # Note this is the only job that leverages amd64
     runs-on: ${{ (inputs.fallback_runner && 'ubuntu-latest') || 'scaleset-jupiterone-infra-amd64' }}
-    container: cypress/browsers:node18.12.0-chrome106-ff106
+    container:
+      # Available containers here: https://hub.docker.com/r/cypress/browsers/tags
+      image: cypress/browsers:latest
+      options: --user 1001
+      # credentials:
```

Comment:
I need to update an ops-platform-job to mass update all the `e2e_trigger.yml` workflows in the frontend repos to pass in `DOCKER_HUB_SRE` as a secret. Once that's done, I can then enable this.